main.py

```diff
@@ -441,13 +442,16 @@ def ui_access_wrapper(self, *args, **kwargs):
         r'/explorationdataextractionhandler', admin.DataExtractionQueryHandler),
     get_redirect_route(r'/frontend_errors', FrontendErrorHandler),
     get_redirect_route(r'/logout', base.LogoutPage),
+    get_redirect_route(
```

Comment:
Add newline above this -- we'll be adding more urls to this block in the future.